libs/ui-components/src/components/form/validations.ts

```
@@ -611,6 +611,59 @@ export const validateMemoryLimit = (memory: string | undefined): boolean => {
+
+// Volume validation schemas
+const singleContainerVolumesSchema = (t: TFunction) =>
+  Yup.array().of(
+    Yup.object().shape({
+      name: volumeNameSchema(t),
+      volumeType: Yup.string()
+        .oneOf([VolumeType.MOUNT_ONLY, VolumeType.IMAGE_MOUNT])
+        .required(t('Volume type is required')),
+      imageRef: Yup.string().when('volumeType', {
+        is: VolumeType.IMAGE_MOUNT,
+        then: () => imageRefSchema(t),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)